web/src/services/locationService.ts

```diff
@@ -44,7 +44,14 @@ const locationService = {
   },
 
   clearQuery: () => {
-    store.dispatch(setQuery({}));
+    store.dispatch(
+      setQuery({
+        tag: undefined,
+        type: undefined,
```

Comment:
There is also a `duration`
```suggestion
        type: undefined,
        duration: undefined,
```